Bottom: cb723ffcde26f51532e852e8fd60eb06f0ac9f49
Top:    ed0505c98af17a6784790f59cac12e3b16031f62
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-15 10:31:41 -0500

Install buildbot plugin dependencies


---
diff --git a/docker/buildbot-master/Dockerfile b/docker/buildbot-master/Dockerfile
index 0947a60c..1baa3f49 100644
--- a/docker/buildbot-master/Dockerfile
+++ b/docker/buildbot-master/Dockerfile
@@ -14,7 +14,11 @@ RUN apk --no-cache add \
 
 # install pip dependencies
 RUN pip --no-cache-dir install \
-    'attrs'
+    'attrs' \
+    'buildbot-console-view' \
+    'buildbot-grid-view' \
+    'buildbot-waterfall-view' \
+    'buildbot-www'
 
 # Create buildbot user
 ARG BUILDBOT_UID=1000
